src/sidebar/app/components/Footer.js

```diff
@@ -286,39 +301,40 @@ class Footer extends React.Component {
               {this.rightText}
             </button>
           </div>
-          <div className="wrapper">
-            <button id="context-menu-button"
-              className="mdl-js-button" />
-            <ul
-              className="mdl-menu mdl-menu--top-right mdl-js-menu context-menu"
-              data-mdl-for="context-menu-button">
-              <li>
-                <button className="mdl-menu__item context-menu-item"
-                   title={browser.i18n.getMessage('exportAsHTML')}
-                   style={{ width: '100%' }}
-                   onClick={ this.exportAsHTML }>
-                  { browser.i18n.getMessage('exportAsHTML') }
-                </button>
-        </li> {
-          !this.state.state.savingLayout && !this.state.state.ignoreChange ?
-              <li>
-                <button className="mdl-menu__item context-menu-item"
-                  title={browser.i18n.getMessage('disableSync')}
-                  style={{ width: '100%' }}
-                  onClick={ this.disconnectFromSync }
-                >
-                  {browser.i18n.getMessage('disableSync')}
-                </button>
-            </li> : null
-        }<li>
-              <button className="mdl-menu__item context-menu-item"
-                 title={browser.i18n.getMessage('feedback')}
-                 style={{ width: '100%' }}
-                 onClick={ this.giveFeedbackCallback }>
-                  { browser.i18n.getMessage('feedback') }
-                </button>
-              </li>
-            </ul>
+
+          <div className="photon-menu open top left" ref={menu => this.menu = menu }>
+            <button id="context-menu-button" onClick={(e) => this.toggleMenu(e)}>
+              <MoreIcon />
+            </button>
+            <div className="wrapper">
+              <ul role="menu" >
+                <li>
+                  <button
```

Comment:
@sebastienbarbier when I was debugging this I noticed that if we remove buttons and just put everything on `<li>` this also works (due to `onClick` being on `<li>` not sure if that simplifies things or not.

Reply:
This trigger errors regarding a10y, we need to have click event on clickable object ike button on link.